src/Shared/Server/infra/log.go

```diff
@@ -20,5 +20,13 @@ func (handler LogHandle) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		log.Fatalln("http server: log handle:", err)
 	}
 
-	handler.logEnable <- (string(body) == "enable")
+	go func() {

```

Comment:
This is more related to the logger, but I'll put it here. The way the logger is setup right now, multiple logging sessions can be started at the same. Either we add ids to "logging sessions" so that multiple logging sessions can be coordinated at the same time or we set it up so that logging is shared between users. I'd go for the first options.